### PR TITLE
Replace QRegExp with QRegularExpression

### PR DESCRIPTION
**Pull request by zneix**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

- Replace QRegExp with QRegularExpression in Args.cpp
- Eliminated a clazy warning in Args.cpp
- Replace QRegExp with QRegularExpression in NuulsUploader.cpp

### CODE REVIEW

File: src/common/Args.cpp [hunk 1]
```diff
@@ -147,7 +147,7 @@ void Args::applyCustomChannelLayout(const QString &argValue)
 
             // Set first tab as selected
             tab.selected_ = window.tabs_.empty();
-            tab.rootNode_ = SplitNodeDescriptor{"twitch", channelName};
+            tab.rootNode_ = SplitNodeDescriptor{{"twitch", channelName}};
```
Comment (Mm2PL) [inline src/common/Args.cpp:150]: Can you please explain this change?

Comment (zneix) [reply src/common/Args.cpp:150]: I explained it in ef292c6's commit message and the PR - Qt Creator thown a warning saying we should wrap this inside braces.

Comment (Mm2PL) [reply src/common/Args.cpp:150]: i see

**Review by pajlada — APPROVED** [review]
Tested and confirmed works :ok_hand:

**Comment by pajlada** [comment]
I'll update the branch and merge this in after https://github.com/Chatterino/chatterino2/pull/2933 is done

**Pull request merged**